rust/qrcode/Makefile

```diff
@@ -1,37 +1,26 @@
 .PHONY: all
-all: build
+all: deploy
 
 .PHONY: node_modules
 .SILENT: node_modules
 node_modules:
 	npm install
 
-.PHONY: build
-.SILENT: build
-build: node_modules
-	dfx canister create --all
-	dfx build
-
-.PHONY: install
-.SILENT: install
-install: build
-	dfx canister install --all
-
-.PHONY: upgrade
-.SILENT: upgrade
-upgrade: build
-	dfx canister install --all --mode=upgrade
+deploy: node_modules
+	dfx deploy
 
 .PHONY: test
 .SILENT: test
-test: install
```

Comment:
`install` doesn't work well for iterative development when the canisters need to upgraded instead of installed.